### ISSUE TITLE
Demo website is not working

### ISSUE THREAD

**Issue by Undeadlol1** [Fraina/Material-Design-ColorPicker#4]
As title states your website does not seem to work

**Comment by fraina** [comment]
hi @Undeadlol1. After checked, I found the demo page works fine with chrome, firefox and safari for windows. This colorpicker is not sopport IE10↓ because css3 animation won't work with older version of IE. I wonder if you are using older version of browser?

**Comment by Undeadlol1** [comment]
Hello Fraina.
I am using latest Chrome browser which is version 46.0.2490.71.
Here is a screenshot of what i see.
![error](https://cloud.githubusercontent.com/assets/4124323/10583609/50eab884-76a7-11e5-8be9-733c57ab9b16.jpg)

**Comment by Undeadlol1** [comment]
Update:
I just noticed the first error and it says that i am using https connection which is not allowed. The problem was in my extension for Chrome called "HTTPS Everywhere". It makes every web connection type https instead of http. I installed it a while ago and completely forgot about it.

I am very ashamed for wasting your time and i want to apologise. Sorry.
I hope you will undearstand me as a fellow developer. I just so used to javascript errors that i forget to read them at all.
